packages/code-du-travail-api/src/server/routes/idcc/index.js

```diff
@@ -0,0 +1,28 @@
+const Router = require("koa-router");
+const API_BASE_URL = require("../api").BASE_URL;
+const routeName = "/idcc";
+const router = new Router({ prefix: API_BASE_URL });
+
+const elasticsearchClient = require("../../conf/elasticsearch.js");
+const getIdccBody = require("./idcc.elastic");
+
+const index =
+  process.env.ELASTICSEARCH_DOCUMENT_INDEX || "code_du_travail_numerique";
```

Comment:
Are we duplicating this config ?
We don't have a external `env` config ?

Reply:
some route/endpoint may use different index name, so i put it on a per routes basis

Reply:
Alright